README.adoc

```diff
@@ -23,4 +23,37 @@ Then write 2 for hdmi_group and 27 for mode for a result of:
 hdmi_group=2 
 hdmi_mode=27 
 . Save the file and reboot the pi.
+ 
+= Compiling and Running the Program
+1. After cloning the repository, `cd` into the "_Installer_" file
+2. Run the command `sudo ./MainInstaller.sh`
+3. Next run the command `make` to create generate the executable for the BackupCamera code
+4. The program should now run with `./BackupCamera`
+   * The program currently takes 6 command line arguments
+      ** 2 for window position (x, y), 2 for window size (x, y) and 2 for camera resolution
+   * If the window is black refer to the _setting up a webcam in the vm_ instructions below
 
+
+= Setting Up a Webcam in the VM
```

Comment:
Can you find someone that uses a Windows VM to verify these steps? They can make comments if the instructions are clear enough or not.